generated/nidcpower/session.py

```diff
@@ -3556,7 +3558,7 @@ def commit(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def configure_digital_edge_measure_trigger(self, input_terminal, edge):
+    def configure_digital_edge_measure_trigger(self, edge, input_terminal=enums.DigitalEdge.RISING):
```

Comment:
You have these off by one errors in many places. The edge is rising, not the input terminal!

Reply:
Fixed